forge-gui/res/cardsfolder/upcoming/escape_tunnel.txt

```diff
@@ -0,0 +1,8 @@
+Name:Escape Tunnel
+ManaCost:no cost
+Types:Land
+A:AB$ ChangeZone | Cost$ T Sac<1/CARDNAME> | Origin$ Library | Destination$ Battlefield | Tapped$ True | ChangeType$ Land.Basic | ChangeNum$ 1 | SpellDescription$ Search your library for a basic land card, put it onto the battlefield tapped, then shuffle.
+A:AB$ Effect | Cost$ T Sac<1/CARDNAME> | ValidTgts$ Creature.powerLE2 | TgtPrompt$ Select target creature with power 2 or less | RememberObjects$ Targeted | ExileOnMoved$ Battlefield | StaticAbilities$ Unblockable | AILogic$ Pump | StackDescription$ {c:Targeted} can't be blocked this turn. | SpellDescription$ Target creature with power 2 or less can't be blocked this turn.
```

Comment:
probably need cleanup

Reply:
Doesn't the` ExileOnMoved$ Battlefield`  already take care of that ?

Reply:
yea that change is wrong, nothing gets remembered on the card